README.rst

```diff
@@ -30,6 +30,10 @@ FAQ
 
   A- The `industrial_ci` still provides valuable checks; it ensures if your package builds without issues. Also installation rules if you define. Just as a headsup that making test cases are highly recommended as your ear may hurt.
 
+- Q- My package uses a custom Point Cloud Library (PCL) version or the [`industrial_calibration`](https://github.com/ros-industrial/industrial_calibration) package, how do I make build work?
+
+  A- You can check [`advanced_industrial_ci`](https://github.com/InstitutMaupertuis/advanced_industrial_ci), this repository provides scripts and binaries for PCL 1.8.0 and the Ceres solver.
```

Comment:
Also,

> this repository

readers might get lost which repository this line refers to (`industrial_ci` or `advanced_industrial_ci`).
You might simply want to replace:

- - this repository
- + which